Source/WebCore/page/Quirks.cpp

```diff
@@ -1756,78 +1756,77 @@ bool Quirks::shouldDisableNavigatorStandaloneQuirk() const
 // to deactivate them for testing.
 bool Quirks::needsIpadMiniUserAgent(StringView host)
```

Comment:
We should probably make  `needsIpadMiniUserAgent` take `URL` instead of `StringView` if we want to drop `equalLettersIgnoringASCIICase`, since `StringView` has no indication about case.